source/Xunit.Gherkin.Quick.UnitTests/GherkinScenarioOutlineTests.cs

```diff
@@ -307,12 +307,578 @@ public void ApplyExampleRow_Digests_Row_Values_Into_Scenario_With_DocString_In_S
+        [InlineData(
+            "I declare: \"<a>< <b>\"",                        // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5< 6\"",                            // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5< -6\"",                          // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a> <= <b>\"",                      // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5 <= 6\"",                          // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5 <= -6\"",                        // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a><=<b>\"",                        // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5<=6\"",                            // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5<=-6\"",                          // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a> <=<b>\"",                       // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5 <=6\"",                           // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5 <=-6\"",                         // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        [InlineData(
+            "I declare: \"<a><= <b>\"",                       // givenExpression
+            "the result for <a> and <b> should be <result>",  // thenExpression
+            "I declare: \"5<= 6\"",                           // givenResult1
+            "the result for 5 and 6 should be True",          // thenResult1
+            "I declare: \"-5<= -6\"",                         // givenResult2
+            "the result for -5 and -6 should be False")]      // thenResult2
+        public void ApplyExampleRow_Digests_Row_Values_Into_Scenario_With_Operators_And_Spaces(
+            string givenExpression,
+            string thenExpression,
+            string givenResult1,
+            string thenResult1,
+            string givenResult2,
+            string thenResult2)
+        {
+            void ValidateStep(Gherkin.Ast.Step step, string keyword, string text, Gherkin.Ast.Step other, bool shouldSubstitute)
```

Comment:
Same applies here - rename arguments, use `expected` and `actual` in names to explain what these are.

Reply:
I have just used the naming of a similar method written before me in other tests as it was approved earlier and my previous stylistic changed were not correct. Gonna change it everywhere then.

Reply:
Done.

Reply:
Thank you!